src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="card image"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">
+        Product code:
+        <span>195434</span>
+      </p>
+      <div class="card__content align">
```

Comment:
```suggestion
      <div class="card__review">
```

Reply:
I reuse card__content in two blocks
 
  &__content {
    display: flex;
    align-items: center;
    justify-content: space-between;
  }
![Screenshot from 2024-09-09 18-30-26](https://github.com/user-attachments/assets/3f0a9313-f6b9-4e0d-be7b-98fdb0da14ed)

Reply:
I get your point - to reuse code. But see the comment below. Also, this element has its own styles in addition, so it's not a problem to duplicate these few properties here.  It's contraversial solution to apply two BEM-element classes to one HTML element. So I'd recommend to get rid of card__content here